src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -300,11 +300,11 @@ You've probably experienced layout shifts due to UI that pops-in at the top or b
 
 If you need to display these types of UI affordances, reserve sufficient space in the viewport for it in advance (for example, using a placeholder or skeleton UI) so that when it loads, it does not cause content in the page to surprisingly shift around.
 
-In some cases adding content dynamically is an integral part of the page. For example, when loading more products to a list of items or when updating live feed content. In those cases you can avoid unexpected layout shifts by:
+In some cases adding content dynamically is important part of user experience. For example, when loading more products to a list of items or when updating live feed content. There are several ways to avoid unexpected layout shifts in those cases:
```

Comment:
I'm not a native speaker, so mainly confirming - isn't it supposed to be ".. is **an** important part of **the** user experience .."?

Reply:
Good catch, it is "an important". No need for "the" next to user experience because it's user experience in a general sense.